Makefile

```
@@ -781,6 +781,33 @@ packaging-macos-bundle:
+# packaging/windows/PACKAGING.md. The recipe is a thin wrapper around
+# scripts/build-windows-bundle.ps1 so per-repo Make-recipe size limits stay
+# satisfied.
+#
+# BUNDLE_GOARCH accepts amd64 (default) or arm64. On arm64 the hook +
+# hook-launcher + mgr still ship as amd64 (documented in the plan and
+# PACKAGING.md); only the gateway varies with BUNDLE_GOARCH.
+WIN_BUNDLE_GOARCH ?= amd64
+WIN_BUNDLE_NAME   := defenseclaw-windows-$(VERSION)-$(WIN_BUNDLE_GOARCH)
+WIN_BUNDLE_DIR    := $(DIST_DIR)/$(WIN_BUNDLE_NAME)
+
+packaging-windows-bundle:
```

checkmake (0.3.2)

[warning] Target body for "packaging-windows-bundle" exceeds allowed length of 5 lines (11).

(maxbodylength)